integration-cli/docker_cli_commit_test.go

```diff
@@ -83,9 +80,7 @@ func (s *DockerSuite) TestCommitHardlink(c *check.C) {
 			break
 		}
 	}
-	if !found {
-		c.Fatalf("Failed to create hardlink in a container. Expected to find %q in %q", inode, chunks[1:])
-	}
+	c.Assert(found, checker.True, check.Commentf("Failed to create hardlink in a container. Expected to find %q in %q", inode, chunks[1:]))
```

Comment:
Your suggestion didn't work exactly....but I did something a bit different to get rid of the for loop. Hope it's acceptable